crates/cli-support/src/lib.rs

```diff
@@ -687,11 +663,7 @@ impl Output {
 
         // And now that we've got all our JS and TypeScript, actually write it
         // out to the filesystem.
-        let extension = if gen.mode.nodejs_experimental_modules() {
-            "mjs"
-        } else {
-            "js"
-        };
```

Comment:
Do Node ES modules not use `mjs`?

Reply:
Both `mjs` and `js` are used: https://nodejs.org/api/packages.html#determining-module-system

None of the other targets use `mjs` even though they all output modules, and the `node` target uses `js` instead of `cjs` even though it is the only non-module output. I can undo this change, it was for consistency more than for correctness. `mjs` would make things easier in a mixed-esm-and-cjs package too.

(Personally I would prefer to rename the `node` output to `cjs` too, but I don't think that's doable in a patch release. You currently have to [rename](https://github.com/Systemcluster/kitoken/blob/wip5/packages/javascript/package.json#L28) it manually when necessary.)

Reply:
Considering this was experimental to begin with, I don't think we have to adhere to Semver.

Reply:
It seems like there is other code that depends on the extension being `.js`, multiple tests fail with `.mjs`. I don't want to spend more time investigating those issues right now.